image/image.go

```diff
@@ -263,3 +263,13 @@ func NewImgJSON(src []byte) (*Image, error) {
 	}
 	return ret, nil
 }
+
+// Check wheather id is a valid image ID or not
+func ValidateID(id string) error {
```

Comment:
Should this be moved to [pkg/stringid](https://github.com/docker/docker/blob/8eee55a35c8161bde7bee50231b67f8e33dd16e0/pkg/stringid/stringid.go) in stead? That's also used to *generate* IDs